<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingswidget</class>
 <widget class="QWidget" name="settingswidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Monitor Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="colorgbox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="forecolorbt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Foreground:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSlider" name="foreopslider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="backgroundSbox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="backgroundcolorbt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="foreSbox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSlider" name="backgroundopslider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="5">
       <widget class="QLineEdit" name="commandTbox"/>
      </item>
      <item row="0" column="2" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="updateintervalSbox">
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="maximum">
           <number>60000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Run this command when clicked:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Update interval:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Geometry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QSpinBox" name="widthSbox">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="marginSbox">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Margin:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="heightSbox">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Display width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Display height:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QCheckBox" name="resizetofitCbox">
        <property name="text">
         <string>Resize to fit panel size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okbt">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok-apply">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backgroundopslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>backgroundSbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>foreopslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>foreSbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundSbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>backgroundopslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>foreSbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>foreopslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="swapbuttongroup"/>
 </buttongroups>
</ui>
